MesaModel/model/SimModel.py

```diff
@@ -117,6 +119,33 @@ def __init__(
                 300,
             )
         )
+        self.ticks_per_home_day = self.model_params.ticks_per_home_day
+
+        if self.speedup > 1:
+            min_ticks = min(self.ticks_per_school_day, self.ticks_per_home_day)
+            # Can't have fewer than 1 tick per school day so reduce the speedup accordingly
+            if self.speedup > min_ticks:
+                self.speedup = min_ticks
+            # Speedup should be divisible by self.ticks_per_school_day
```

Comment:
These comments might be handy in user docs too. I think the CLI in the main README doesn't mention the `--speedup` arg yet.